logs-shipper/build.gradle

```diff
@@ -44,4 +53,15 @@ shadowJar {
     transform(com.github.jengelman.gradle.plugins.shadow.transformers.Log4j2PluginsCacheFileTransformer)
 }
 
+task start(type: Exec) {
```

Comment:
I think it makes more sense this task will be in the root dir (which means the root `build.gradle`), as it relates to the entire system (haproxy, logs-shipper, elasticsearch)
Then you don't need to do this "cd.."

Reply:
true :)